src/lib/components/Pagination/PaginationButton.tsx

```diff
@@ -44,4 +49,32 @@ const PaginationButton: FC<PaginationButtonProps> = ({
 };
 
 PaginationButton.displayName = 'Pagination.Button';
-export default PaginationButton;
+
+export const PaginationNavigation: FC<PaginationPrevButtonProps> = ({
+  children,
+  className,
+  onClick,
+  theme: customTheme = {},
+  disabled,
```

Comment:
Maybe `disabled = false,`?

Reply:
@rluders Thanks for noticing, missed it. Fixed now.